### ISSUE TITLE
Meteor 1.1 (and later)

### ISSUE THREAD

**Issue by timothyarmes** [strikeout/mylar#8]
I see that the latest merges with the core packages were for Meteor 1.0. Do you intend to provide support for Meteor 1.1 (and the upcoming 1.2) versions?

**Comment by strikeout** [comment]
Yes, I will merge the latest Meteor upstream changes soon(ish) tho I'm a bit short on time these days so pull-requests are also welcome :)

**Comment by sunny-g** [comment]
@timothyarmes my PR just got merged so you now have support for Meteor 1.1!

**Comment by tacone** [comment]
:+1:

**Comment by mankash** [comment]
Hello guys - Please see this issue: https://github.com/strikeout/mylar/issues/10

How are you able to perform encrypted search with the above issue blocking this?

Thanks
